dot/sync/message_test.go

```diff
@@ -36,6 +37,16 @@ func addTestBlocksToState(t *testing.T, depth int, blockState BlockState) {
 	}
 }
 
+func TestMain(m *testing.M) {
+	wasmFilePaths, _ := runtime.GenerateRuntimeWasmFile()
```

Comment:
maybe check the error? if it fails to fetch the wasm blob then it should it exit

same with other funcs